bucket/jabref.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "5.1",
+    "description": "Free Reference Manager",
```

Comment:
Completely saying-nothing description.

What references?? File? Dependencies? Software?

Reply:
Updated in new commit.

Reply:
Updated. See commit "Update jabref.json"